js/__fixtures__/credentials/bbs/credentialTemplate.json

```diff
@@ -0,0 +1,28 @@
+{
+    "@context": [
+      "https://www.w3.org/2018/credentials/v2",
```

Comment:
I believe we're going to make this `https://www.w3.org/2021/credentials/v1`

Reply:
I am keeping this until the correct one is provisioned, I also find the date style versioning to be less than intuitive....